dashboard/app/models/activity_section.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: activity_sections
+#
+#  id                 :integer          not null, primary key
+#  lesson_activity_id :integer          not null
+#  position           :integer          not null
+#  properties         :string(255)
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+
+class ActivitySection < ApplicationRecord
+  include SerializedProperties
+
+  belongs_to :activity, class_name: 'LessonActivity', foreign_key: :lesson_activity_id, inverse_of: :activity_sections
+  has_one :lesson, through: :activity
+
+  has_many :script_levels
+
+  serialized_attrs %w(
+    display_name
+    remarks
+    slide
+    text
+    tips
```

Comment:
@davidsbailey I know I used tips so you probably were just sticking to that. Do you think it is specific enough? I know you commented on something else where I just called it tips

Reply:
good question Dani... I was confused in the context of the react code because there are also tooltips, and a I wasn't sure whether a react component with `tip` in the name was talking about lesson tips or tool tips. I think this name is ok with me in the data model if it is ok with you.

Reply:
ok great lets stay with tips then